runtest.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+set -e
+
+export NODE_ENV=test
+export BLUEBIRD_DEBUG=1
```

Comment:
What's this for? Seems like this module is no longer even using bluebird.

Reply:
Oops, just copypasta. Will delete.

Reply:
I don't think this was for this module's use of bluebird. It was to enable better error details in our tests in our apps/modules that use bluebird(?)

Reply:
That's true.

Reply:
Details: http://bluebirdjs.com/docs/api/environment-variables.html